js/foam/node/handlers/FileHandler.js

```diff
@@ -47,10 +57,23 @@ CLASS({
       if ( this.url.parse(req.url).pathname !== this.pathname ) return false;
 
       this.fs.readFile(this.file, function(err, data) {
```

Comment:
Yes, I'm not sure we use the FileHandler anymore.

On Tue, Nov 24, 2015 at 2:48 PM adamvy <notifications@github.com> wrote:

> In js/foam/node/handlers/FileHandler.js
> <https://github.com/foam-framework/foam/pull/475#discussion_r45784971>:
>
> > @@ -47,10 +57,23 @@ CLASS({
> >        if ( this.url.parse(req.url).pathname !== this.pathname ) return false;
> >
> >        this.fs.readFile(this.file, function(err, data) {
>
> Might be better to restructure this as
>
> this.fs.createReadStream(this.file) similar to the StaticFileHandler
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/foam-framework/foam/pull/475/files#r45784971>.
>